packages/kakoune.rb

```diff
@@ -0,0 +1,26 @@
+require 'package'
+
+class Kakoune < Package
+  # TODO prebuild version
+  description 'mawww\'s experiment for a better code editor'
+  homepage 'http://kakoune.org/'
+  version 'v2018.09.04'
+  source_url 'https://github.com/mawww/kakoune/releases/download/v2018.09.04/kakoune-2018.09.04.tar.bz2'
+  source_sha256 '7a31c9f08c261c5128d1753762721dd7b7fe4bb4e9a3c368c9d768c72a1472e1'
+
+  depends_on 'ncurses'  => :build
+  depends_on 'asciidoc' => :build
+  depends_on 'libxslt'  => :build
+
+  def self.build
+    Dir.chdir("src") do
+      system "make", "debug=no"
```

Comment:
You generally want to add configure options `--prefix=#{CREW_PREFIX}` and `--libdir=#{CREW_LIB_PREFIX}`.  Does this archive include a configure script?

Reply:
I examined the codebase and it looks like you could do something like this:
```
system "make PREFIX=#{CREW_PREFIX} debug=no"
```

Reply:
My bad, kakoune's Makefile has `PREFIX` already set to `/usr/local`, so I didn't notice.

>Does this archive include a configure script?

As far as I know - no.